api-report/merge-tree.api.md

```diff
@@ -539,9 +537,6 @@ export interface IMergeTreeTextHelper {
     getText(refSeq: number, clientId: number, placeholder: string, start?: number, end?: number): string;
 }
 
-// @public (undocumented)
-export function internedSpaces(n: number): string;
```

Comment:
Should this be deprecated in `main`? Do partners still consume `merge-tree`?

Reply:
they consume the package, but not the class. basically if it impacts the api-report or the type tests it should be deprecated in main before removal.